apps/desktop/.env.example

```diff
@@ -1,6 +1,6 @@
-VITE_ANALYTICS_WRITE_KEY_PROD=
```

Comment:
Are these env vars just used by the Tauri app? if not then we should move this up to the top level of the repo.

Reply:
moved all env to root